install/src/HieInstall.hs

```diff
@@ -91,6 +91,7 @@ defaultMain = do
       )
 
     -- stack specific targets
+    when isRunFromStack (phony "stack-install-cabal" (need ["cabal"]))
```

Comment:
"stack-install-cabal" does not fit to the help-message, I think? Or am I overlooking something?

Reply:
The `stack-` prefix is added by `stackTarget` (it could be added to cabal targets and it would be `cabal-install-cabal` 😆)

Reply:
I see, thanks for the clarification!